### ISSUE TITLE
MacOS Monterey 12.01 (Intel): Electron download (any version) fails to complete due to permission error

### ISSUE THREAD

**Issue by wykananda** [electron/fiddle#922]
Installed the latest version of Electron Fiddle Version 0.27.2 (15.0.0) on MacOS Monterey in the Applications directory.
On first run, application starts up but fails to install Electron. The download appears to complete but "Checking status" icon spins forever and the following error is logged in the console. This error occurs regardless of which Electron version is selected/downloaded. 
> ...
> Binary: Version 16.0.4 download progress: 1
> instrument.ts:129 Binary: Failure to download 16.0.4 Error: EACCES: permission denied, mkdir '/Users/wyk/Library/Caches/electron/625969703fca60f417f168de15e05663448414da45e461156402c401d6399e8d'

**Comment by wykananda** [comment]
I was able to work around the issue by manually editing permissions on the /Users/{user}/Library/Caches/electron folder adding the {user} with read and write privileges. But this does seem like a bug, no?

**Comment by VerteDinde** [comment]
Hey @wykananda, thanks for reporting this issue! I'm also on Monterey, but unfortunately, I can't repro this issue. Would you mind seeing if uninstalling and reinstalling Fiddle resolves the issue for you? It's possible that this is a permissions issue that came from upgrading from Big Sur; I'll look into this a bit more to see if I can reproduce.

**Comment by wykananda** [comment]
Hi there. I did upgrade from Big Sur - so perhaps that's the cause.  I deleted the /Users/{user}/Library/Caches/electron folder and the Fiddle app and then reinstalled it. This time updating the Electron build from within the Fiddle app worked fine - the /Users/{user}/Library/Caches/electron folder showed up with {user} Read&Write permissions and Electron versions downloaded, unzipped, and installed just fine from within Fiddle.
